.vscode/launch.json

```diff
@@ -8,20 +8,20 @@
 			"name": "Launch Client",
 			"runtimeExecutable": "${execPath}",
 			"args": ["--extensionDevelopmentPath=${workspaceRoot}"],
-			"outFiles": ["${workspaceRoot}/out/client/**/*.js"],
-			"preLaunchTask": {
-				"type": "npm",
-				"script": "compile"
-			}
+			"outFiles": ["${workspaceRoot}/client/out/**/*.js"],
+			"sourceMaps": true,
+			"preLaunchTask": "npm: watch" //The new vscode window still needs to be reloaded after changes (Ctrl+Shift+F5 in the debugger)
```

Comment:
Are comments allowed here?

Reply:
Yes, comments are allowed in all VS Code configuration files.